static-bundles.json

```diff
@@ -921,9 +921,11 @@
     },
     {
       "files": [
+        "protocol/js/protocol-modal.js",
```

Comment:
YAY! Hope that wasn't hard to figure out :) I think the "new way" of gulp copying in the resources from the npm package is working quite well, but I'd love to hear any feedback you have after working with it.